### ISSUE TITLE
Redirection issue with Magx cookie

### ISSUE THREAD

**Issue by Landemarre** [freemint/freemint#115]
in proc.c function new_proc() toswin2 look if Magic is present or not to not do Fforce(-1, ..), if we add magx cookie for some software compatibility under Mint then some softwares such as bash can not be correctly redirect in window. 
We should rather look at Mint cookie to do Fforce(-1) or not ?

**Comment by th-otto** [comment]
Why should we add magx cookie in mint? Or did i misunderstand something?

**Comment by mikrosk** [comment]
No, I get it. @Landemarre is referring to toswin2 source code. I don't see a problem with reverting the condition (!gl_magx -> gl_mint) but you should test such change first.

**Comment by Landemarre** [comment]
Hello

I just tested change, rather replace !gl_magx for gl_mint I have put this condition:
if ((!gl_magx) || (gl_mint))
			Fforce(-1, kidfd);

It work perfectly

Olivier

**Comment by mikrosk** [comment]
Btw how do you add that magx cookie? Because now I think about it, I have never heard about anyone doing it, at least I don't know a tool "add your own system cookies". There is that N.AES hack in xaaes.cnf but magx cookie in mint? What software needs it?

**Comment by Landemarre** [comment]
Hello
MyAES is able to add magx cookie as NAES (don't know if it usefull in this case) it's only an option in this 2 case. For magx it is interesting for Papyrus software, this is the only way I know to be able to display color icons!

**Comment by Landemarre** [comment]
I think since Magic 4 handle -1 should be redirected too
http://toshyp.atari.org/en/005009.html#Fforce

**Comment by xdelatour** [comment]
> Btw how do you add that magx cookie? Because now I think about it, I have never heard about anyone doing it, at least I don't know a tool "add your own system cookies". There is that N.AES hack in xaaes.cnf but magx cookie in mint? What software needs it?

Helmut tried to add MagiC cookie to XaAES (not MiNT) many years ago, commit 35b4ed725c4833c7d7c41673e7f102b05720c928

Not sure it's useful especially since Sconfig doesn't exist in MiNT

**Comment by mikrosk** [comment]
Hmm, this whole issue looks like it has been solved by itself? I mean, if Magic 4 already supports Fforce(-1, ..), we should probably just remove [that check](https://github.com/freemint/toswin2/blob/master/proc.c#L148) from TosWin2 completely and let it be as is?

**Comment by xdelatour** [comment]
The related code doesn't exist anymore. I can't find when it was deleted
